byron/chain/executable-spec/src/Cardano/Spec/Chain/STS/Block.hs

```diff
@@ -70,10 +71,16 @@ data BlockBody
   -- ^ Protocol version
   } deriving (Generic, Show)
 
-instance HasTypeReps BlockBody
-
 makeLenses ''BlockBody
 
+instance HasTypeReps BlockBody where
```

Comment:
Why do we need the custom instance here?

Reply:
good question, we _don't_, I've removed it again